pkgs/tools/networking/curl-impersonate/deps.nix

```diff
@@ -0,0 +1,29 @@
+# Generated by update.sh
+{ fetchurl }:
+
+{
+  "curl-7.84.0.tar.xz" = fetchurl {
```

Comment:
This version has 17 known CVEs https://curl.se/docs/security.html

Are we just going to ignore them or do we want to mark them in meta.knownVulnerabilties?

Reply:
I guess it is safe to discard most of them based on the use case (only the protocols the imitated browsers support have any relevance, and I'd argue the use case is even outside the prime usefulness of HSTS support), but indeed there are some unfortunate things, like today's CVE-2023-32001 (well it also applies to our main `curl`, we'll see how long it takes for the fix to reach the channel).

Reply:
I'm fine with adding `meta.knownVulnerabilities` if they are relevant to the build configuration options curl-impersonate uses. I'll investigate and add any relevant ones today or tomorrow

Reply:
The only relevant ones for how curl is built that are worth a `meta.knownVulnerabilties` that I could find from https://curl.se/docs/vuln-7.84.0.html are:
* CVE-2023-32001 - fopen TOCTOU, CVSSv3 base score 5.5
* CVE-2022-43551 - HSTS bypass, CVSSv3 base score 7.5
* CVE-2022-42916 - HSTS bypass, CVSSv3 base score 7.5

I'll push in a minute to update this PR for this and the removal of `pytest` from `PYTHONPATH`